fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/common/BulkDataUtils.java

```diff
@@ -112,6 +112,8 @@ public static PartETag multiPartUpload(AmazonS3 cosClient, String bucketName, St
             log("multiPartUpload", "Upload part Error - " + sdke.getMessage());
             cosClient.abortMultipartUpload(new AbortMultipartUploadRequest(bucketName, itemName, uploadID));
             throw sdke;
+        } finally {
+            dataStream.close();
```

Comment:
This feels like a side-effect to me...I don't think the method should close the stream...that should be the responsibility of the caller.

Reply:
I agree with @punktilious

Reply:
@punktilious haha, thats what I said while we were on the call :-)
will create a separate close method and do it there

Reply:
ok, I'll take this back out.  presently we're always calling this method with a ByteArrayInputStream and so the close doesn't do anything either